FinSearchUnified/BusinessLogic/Export.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace FinSearchUnified\BusinessLogic;
+
+use FinSearchUnified\Bundles\SearchBundle\Condition\HasActiveCategoryCondition;
+use FinSearchUnified\Bundles\SearchBundle\Condition\HasActiveChildCategoryOfCurrentShopCondition;
+use FinSearchUnified\XmlInformation;
+use Shopware\Bundle\SearchBundle\Condition\IsAvailableCondition;
+use Shopware\Bundle\SearchBundle\Criteria;
+use Shopware\Bundle\SearchBundleDBAL\QueryBuilder;
+use Shopware\Bundle\SearchBundleDBAL\QueryBuilderFactory;
+use Shopware\Bundle\StoreFrontBundle\Service\ContextServiceInterface;
+use Shopware\Models\Config\Value;
+use Shopware\Models\Shop\Repository;
+use Shopware\Models\Shop\Shop;
+use UnexpectedValueException;
+
+class Export
+{
+    /**
+     * @var Shop $shop
```

Comment:
Mentioning the variable names shouldn't be necessary. I think just giving the type is sufficient.